sync/src/main/java/tech/pegasys/teku/sync/PeerSync.java

```diff
@@ -46,16 +49,31 @@
   private final BlockImporter blockImporter;
 
   private final AsyncRunner asyncRunner;
+  private final Counter blockImportSuccessResult;
+  private final Counter blockImportFailureResult;
 
   private volatile UnsignedLong startingSlot = UnsignedLong.valueOf(0);
 
   public PeerSync(
       final AsyncRunner asyncRunner,
       final RecentChainData storageClient,
-      final BlockImporter blockImporter) {
+      final BlockImporter blockImporter,
+      final MetricsSystem metricsSystem) {
     this.asyncRunner = asyncRunner;
     this.storageClient = storageClient;
     this.blockImporter = blockImporter;
+
+    this.blockImportSuccessResult =
+        metricsSystem.createCounter(
+            TekuMetricCategory.BEACON,
+            "block_import_success_result",
+            "The number of block imports that have been successfully completed");
+
+    this.blockImportFailureResult =
+        metricsSystem.createCounter(
+            TekuMetricCategory.BEACON,
+            "block_import_failure_result",
+            "The number of block imports that have failed");
```

Comment:
Can we use a single labelled counter for this instead:
```suggestion
    final LabelledMetric<Counter> blockImportCounter =  metricsSystem.createLabelledCounter(
            TekuMetricCategory.BEACON,
            "block_import_total",
            "The number of block imports performed",
            "result");
    this.blockImportSuccessResult = blockImportCounter.labels("imported");
    this.blockImportFailureResult = blockImportCounter.labels("rejected");
```

That way it's easy to show both these metrics on one graph or sum them together.